Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM openjdk:18-jdk-slim
+
+ENV JAVA_HOME /usr/local/openjdk-18
+ENV PATH $JAVA_HOME/bin:/usr/share/maven/bin:$PATH
+
+RUN apt-get update && \
+    apt-get install -y maven && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+COPY .env .
+COPY pom.xml .
+COPY ./src ./src
+COPY ./__tests__ ./__tests__
+
+RUN mvn clean install
+
+RUN mvn install:install-file -Dfile=./target/voucherify-java-sdk-12.0.0.jar -DgroupId=local -DartifactId=voucherify-java-sdk -Dversion=12.0.0 -Dpackaging=jar
```

Comment:
Tutaj nie wiem, czy nie warto tego zrobić tego tak, żeby te wersje się zaciągały z jednego miejsca. Niż tak zmieniać w 5 plikach wersje.

Reply:
trudna sprawa :)

Reply:
Część rzeczy jest połapanych w mustache, a resztę myślę, że najłatwiej i wcale nie problemowo złapać używając `znajdź w projekcie` - 12.0.0.
Podmienienie tego by było to brane z jakiegoś .env to powiedziałbym, że może być częścią improvmenta przy okazji wypuszczenia managmant API w najbliższym czasie.